mail_tracking_mass_mailing/views/mail_mail_statistics_view.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<openerp>
+<data>
+
+<record model="ir.ui.view" id="view_mail_mail_statistics_form">
+    <field name="name">Add tracking email info</field>
+    <field name="model">mail.mail.statistics</field>
+    <field name="inherit_id" ref="mass_mailing.view_mail_mail_statistics_form"/>
+    <field name="arch" type="xml">
+        <field name="res_id" position="after">
+            <field name="tracking_state"/>
+        </field>
+        <xpath expr="//form" position="inside">
+            <group>
+                <field name="mail_tracking_id" />
+            </group>
+            <label for="tracking_event_ids"/>
+            <div>
+                <field name="tracking_event_ids">
```

Comment:
Because in mail_tracking there is no tree view for this field.
Here mail_tracking use a custom definition too: https://github.com/OCA/social/blob/d3870e4c78b734eb12cccfc732474001a43bd421/mail_tracking/views/mail_tracking_email_view.xml#L46